OpenSim/Examples/ControllerExample/tugOfWar_model_ThelenOnly.osim

```diff
@@ -281,10 +281,6 @@
 				<Thelen2003Muscle name="muscle1">
 					<!--Flag indicating whether the force is disabled or not. Disabled means that the force is not active in subsequent dynamics realizations.-->
 					<isDisabled>false</isDisabled>
-					<!--Minimum allowed value for control signal. Used primarily when solving for control values.-->
```

Comment:
The desired user interaction (and behind-the-scenes behavior) isn't so clear. This scenario is also relatively unlikely to occur, so I'd suggest adding this to the list of tests we will add post-beta to avoiding holding up this PR.